source/benchmarking/odm-benchmarking.md

```diff
@@ -0,0 +1,393 @@
+
+Dataset: The dataset (LARGE_DOC_NESTED) is contained within `large_doc_nested.json` and consists of a sample document
+stored as strict JSON with an encoded length of approximately 8,000 bytes.
+
+Dataset size: For scoring purposes, the dataset size is the size of the `large_doc_nested` source file (8,000 bytes)
+times 10,000 operations, which equals 80,000,000 bytes or 80 MB.
+
+| Phase       | Description                                                                                                                                                                                                                                                  |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Setup       | Load the LARGE_DOC_NESTED dataset into memory as an ODM-appropriate model object. Insert 10,000 instances into the database, saving the value of the `unique_id` field for the first item in each model's `embedded_str_doc_array` nested model into a list. |
+| Before task | n/a.                                                                                                                                                                                                                                                         |
+| Do task     | For each of the 10,000 `unique_id` values, perform a filter operation to search for the parent LARGE_DOC_NESTED model.                                                                                                                                       |
+| After task  | n/a.                                                                                                                                                                                                                                                         |
+| Teardown    | Drop the collection associated with the LARGE_DOC_NESTED model.                                                                                                                                                                                              |
+
+## Benchmark platform, configuration and environments
+
+### Benchmark Client
+
+The benchmarks should be run with the most recent stable version of the ODM and the newest version of the driver it
+supports.
+
+### Benchmark Server
+
+The MongoDB ODM Performance Benchmark must be run against a MongoDB replica set of size 1 patch-pinned to the latest
+stable database version without authentication or SSL enabled. This database version should be updated as needed to
+better differentiate between ODM performance changes and server performance changes. The Benchmark should be run on the
+established internal performance distro for the sake of consistency.
```

Comment:
Should "established internal performance distro" reference another document. Based on @dariakp's resolved comment above, I assume this is referring to the basic driver benchmarks.